core/src/com/unciv/models/ruleset/unique/UniqueType.kt

```diff
@@ -476,6 +476,8 @@ enum class UniqueType(val text: String, vararg targets: UniqueTarget, val flags:
     CarryAirUnits("Can carry [amount] [mapUnitFilter] units", UniqueTarget.Unit),
     CarryExtraAirUnits("Can carry [amount] extra [mapUnitFilter] units", UniqueTarget.Unit),
     CannotBeCarriedBy("Cannot be carried by [mapUnitFilter] units", UniqueTarget.Unit),
+    // Used only as mapUnitFilter marker for CarryAirUnits and CarryExtraAirUnits, has no actual implementation
```

Comment:
In that case this shouldn't be a unique here then

Reply:
Why replace? Keep it as is, but not as a typed unique. Working as intended

Reply:
We can have this one untyped, because it's working as intended - it's not harcoded referenced anywhere, it's a *filtering* unique :D

Reply:
That last one